frontend/src/utils/templateActions.js

```diff
@@ -25,37 +27,39 @@ export const actionIcon = a => {
   return <DownloadOutlined />;
 };
 
-export const actionsToButtonList = (urlBase, actions, fullWidth=false) =>
+export const actionsToButtonList = (urlBase, actions, fullWidth = false) =>
   (actions.items || []).map((a) =>
     <Link key={a.id.toString()} to={`${urlBase}/actions/${a.id}/`}>
-      <Button icon={actionIcon(a)} {...(fullWidth ? {style: {width:"100%"}} : {})}>{a.name}</Button>
+      <Button icon={actionIcon(a)} {...(fullWidth ? { style: { width: "100%" } } : {})}>{a.name}</Button>
     </Link>
   );
 
-export function ActionDropdown({urlBase, actions, fullWidth = true}) {
+export function ActionDropdown({ urlBase, actions, fullWidth = true }) {
   const history = useNavigate();
   const actionMenu = (
     <Menu>
-      { actions.items ? actions.items.map((a) =>
-          <Menu.Item key={a.id.toString()}>
-            <Button
-              icon={actionIcon(a)}
-              onClick={() => history(`${urlBase}/actions/${a.id}/`)}
-              {...(fullWidth ? {style:{width:"100%", border:0, textAlign:'left'}} : {style:{border:0}})}
-            >
-              {a.name}
-            </Button>
-          </Menu.Item>) :
-          <Menu.Item>Loading ...</Menu.Item>
+      {actions.items ? actions.items.map((a) =>
+        <Menu.Item key={a.id.toString()}>
+          <Button
+            icon={actionIcon(a)}
+            onClick={() => history(`${urlBase}/actions/${a.id}/`)}
+            {...(fullWidth ? { style: { width: "100%", border: 0, textAlign: 'left' } } : { style: { border: 0 } })}
+          >
+            {a.name}
+          </Button>
+        </Menu.Item>) :
+        <Menu.Item>Loading ...</Menu.Item>
```

Comment:
I'm pretty sure that this code is creating one "Loading..." menu item but the user will never see it because we don't render the Dropdown if the actions.items array is empty, so adding the Loading item is redundant.
It might be better to just have a single check for `actions.items.length > 0` at the top of the function. If it's empty just return null. If it has items then create the menu and render the Dropdown.